spec/support/rbac_seed_context.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.shared_context "rbac_seed_objects" do
+  let(:app_name) { 'catalog' }
+  let(:resource) { "portfolios" }
+  let(:permissions) { ["#{app_name}:#{resource}:read"] }
+  let(:resource_id1) { "99" }
```

Comment:
This could be removed and replaced inline on line 13 I think, there's nothing checking this value so seems superfluous to promote it to a `let`